source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md

```diff
@@ -0,0 +1,375 @@
+
+What makes your site code unique is your selection of contributed modules and themes, and any custom modules or themes your development team has created. These customizations need to be replicated in your new project structure.
+
+### Composer packages
+
+1. Copy your package list from the "requires" section of your site's composer.json and add it to the new site's composer.json. If your composer.json defines additional repositories or patches, copy those over too. Take care not to overwrite the `upstream-configuration` package & repository.
+
+  Note: if your old site has custom patches in its codebase, make sure to copy those over as well.
+
+1. `composer update`
+
+1. `git status` to see if files have been added that aren't ignored by `.gitignore`. If anything shows up other than `composer.*`, add it to `.gitignore` until `git status` only shows the composer files being modified.
+
+1. `git add composer.*; git commit -m "Add composer packages"`
+
+  <Accordion title="Repositories and patches in composer.json" id="repositories-and-patches-in-composer-json" icon="info-sign">
+
+  Repository in old site's `composer.json`
+
+  ![repository in old composer.json](https://i.imgur.com/hO0snBW.png)
+
+  Copied to new composer.json without disturbing the "upstream-configuration" one that was already there:
+
+  ![repository moved to new composer.json](https://i.imgur.com/T6eNnXj.png)
+
+  Patches:
+
+  ![repository moved to new composer.json](https://i.imgur.com/x2SYPb1.png)
+
+  </Accordion>
+
+### Custom Code
+
+Manually copy custom code from the old site to the corresponding Pantheon site directory.
+
+- `web/modules/custom` and `web/libraries` are ignored, so `git add -f`
+
+#### Modules and Themes
+
+[Pantheon site folder] Modules:
+
+```bash{promptUser:user}
+cp -R ../old-site/modules/custom web/modules
+git add web/modules/custom
+git commit -m "Copy custom modules"
+```
+
+[Pantheon site folder] Themes:
+
+```bash{promptUser:user}
+cp -R ../old-site/themes/custom web/themes
+git add web/themes/custom
+git commit -m "Copy custom themes"
+```
+
+Follow suit with any other custom code you need to carry over.
+
+#### settings.php
+
+Your existing site may have customizations to `settings.php` or other configuration files. Review these carefully and extract relevant changes from these files to copy over. Always review any file paths referenced in the code, as these paths may change in the transition.
+
+We don't recommend that you completely overwrite the `settings.php` file with the old one, as it contains customizations for moving the configuration directory you don't want to overwrite, as well as platform-specific customizations.
+
+The resulting `settings.php` should have no `$databases` array.
+
+### Configuration
+
+Copy over exported configuration from the original site. From the Pantheon D9 site, run the following commands:
+
+  ```bash{promptUser: user}
+  mkdir config
+  git mv sites/default/config/* config/
```

Comment:
Is it safe enough to assume config would be under sites/default/config? Maybe a more generic instruction here will be better? ("Copy over your configuration folder")

Reply:
@kporras07 can we mark this as resolved? https://github.com/pantheon-systems/documentation/pull/6862/files#diff-35934f1f5bef5acdfd3db6f375e828dd4aea956210830606d3a7eee454e5374fR166-R174